src/services/article.ts

```diff
@@ -1,18 +1,38 @@
 import { Article } from "../types/article";
 import { ArticleRepository } from '../repositories/articles';
+import articleRepository from '../repositories/articles';
+import {getArticleManuscript} from '../xml-exporter/article-parser';
+import {applyChangesToManuscript} from '../model/changes.utils';
+import changeRepository from "../repositories/changes";
+import {JSONObject} from "../model/manuscript";
+import {serializeManuscript} from "../xml-exporter/manuscript-serializer";
 
 export type ArticleService = {
   getArticles: (page: number) => Promise<Array<Article>>;
   findByArticleId: (articleId: string) => Promise<Article>;
+  exportXml: (articleId: string) => Promise<string | null>;
 }
 
 export default (articleRepo: ArticleRepository): ArticleService => {
```

Comment:
`changesRepo` needs injecting here

Reply:
You'll also need to pass a `mockChangesRepo` to the unit test instances of this to get that to pass ci